File: VB/XPOReport/Customer.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Xpo

Namespace XPOReport
	Public Class Customer
		Inherits XPObject
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub

		Private _name As String
		Public Property Name() As String
			Get
				Return _name
			End Get
			Set(ByVal value As String)
				SetPropertyValue("Name", _name, value)
			End Set
		End Property

		Private _age As Integer
		Public Property Age() As Integer
			Get
				Return _age
			End Get
			Set(ByVal value As Integer)
				SetPropertyValue("Age", _age, value)
			End Set
		End Property
	End Class
End Namespace

File: CS/XPOReport/Customer.cs
using System;
using DevExpress.Xpo;

namespace XPOReport {
    public class Customer : XPObject {
        public Customer(Session session)
            : base(session) {
        }

        string _name;
        public string Name {
            get { return _name; }
            set { SetPropertyValue("Name", ref _name, value); }
        }

        int _age;
        public int Age {
            get { return _age; }
            set { SetPropertyValue("Age", ref _age, value); }
        }
    }
}